include/cbl/CBLReplicator.h

```diff
@@ -66,10 +67,20 @@ _cbl_warn_unused
 CBLAuthenticator* CBLAuth_CreatePassword(FLString username, FLString password) CBLAPI;
 
 /** Creates an authenticator using a Couchbase Sync Gateway login session identifier,
-    and optionally a cookie name (pass NULL for the default.) */
+    and optionally a cookie name (pass NULL for the default.)
+@note You are responsible for freeing the returned reference. */
 _cbl_warn_unused
 CBLAuthenticator* CBLAuth_CreateSession(FLString sessionID, FLString cookieName) CBLAPI;
 
+#ifdef COUCHBASE_ENTERPRISE
+
+/** Create an authenticator that presents a client certificate to the server during the initial SSL/TLS
```

Comment:
Here, an `s` is missing. 🤓

```suggestion
/** Creates an authenticator that presents a client certificate to the server during the initial SSL/TLS
```